tests/sentry/incidents/endpoints/test_project_alert_rule_index.py

```diff
@@ -541,3 +541,134 @@ def test_offset_pagination(self):
 
         self.assert_alert_rule_serialized(self.two_alert_rule, result[0], skip_dates=True)
         self.assert_alert_rule_serialized(self.yet_another_alert_rule, result[1], skip_dates=True)
+
+
+@freeze_time()
+class AlertRuleCreateEndpointTestCrashRateAlert(APITestCase):
+    endpoint = "sentry-api-0-project-alert-rules"
+    method = "post"
+
+    def setUp(self):
+        super().setUp()
+        self.valid_alert_rule = {
+            "aggregate": "crash_free_percentage()",
+            "query": "",
```

Comment:
Will queries for session alerts always just be an empty string?

Reply:
I am not entirely sure yet at this point but possibly yes